File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej4sec.py
# Programa: ej4sec.py
# Propósito: Dados dos números, mostrar la suma, resta, división y multiplicación de ambos.
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * numero1
#   * numero2
#   * suma: num1+num2
#   * resta: num1-num2
#   * multiplicación: num1*num2
#   * división: num1/num2
#
# Algoritmo:
#   LEER numero1, numero2
#   suma <-- numero1 + numero2
#   resta <-- numero1 - numero2
#   multiplicación <-- numero1 * numero2
#   división <-- numero1 / numero2
#   ESCRIBIR suma
#   ESCRIBIR resta
#   ESCRIBIR multiplicacion
#   ESCRIBIR division


# Petición de datos
numero1 = float(input("Introduce el numero1: "))
numero2 = float(input("Introduce el numero2: "))
print("-------------------------------------------")

#Cálculos
suma = (numero1+numero2)
resta = (numero1-numero2)
multiplicacion = (numero1*numero2)
division = (numero1 / numero2)

# Salida
print("La suma de ambos numeros es", suma)
print("La resta de ambos numeros es", resta)
print("La multiplicación de ambos numeros es", multiplicacion)
print("La división de ambos numeros es", division)

File: primerTrimestre/java basico/1secuenciales/Ej4sec.java
/**Dados dos números, mostrar la suma, resta, división y multiplicación de ambos.
 * 
 * @author Antonio Quesada
 */

import java.util.Scanner;

public class Ej4sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    double num1;
    double num2;
    
    //Pedimos que introduzca un número y lo lea.
    System.out.println("Introduce el primer número: ");
    num1 = s.nextDouble();
    System.out.println("Introduce el segundo número: ");
    num2 = s.nextDouble();
    
    //Imprimimos y calculamos las operaciones sin utilizar variables para mostrarlo.
    System.out.println("La suma es "+(num1+num2));
    System.out.println("La resta es "+(num1-num2));
    System.out.println("La multiplicación es "+(num1*num2));
    System.out.println("La división es "+(num1/num2));
    
        
    

  }

}
